src/log-store/src/raft_engine/log_store.rs

```diff
@@ -33,7 +33,7 @@ use crate::error::{
 };
 use crate::raft_engine::protos::logstore::{EntryImpl, NamespaceImpl as Namespace};
 
-const NAMESPACE_PREFIX: &str = "__sys_namespace_";
+const NAMESPACE_PREFIX: &str = "$$_sys_namespace_";
```

Comment:
Is this introducing breaking change?

Reply:
It is. https://github.com/GreptimeTeam/greptimedb/pull/1998 should be merged first @v0y4g3r

Reply:
Is this change necessary for windows platforms?

Reply:
No, it's necessary for upgrading raft-engine, and it's the the new raft-engine that supports Windows.